syz-cluster/dashboard/handler.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
```

Comment:
The proposal is to minimize the amount of files we're adding to the package "main".
Every new file in "main" package require manual efforts during the github repo downstreaming.

Reply:
The `main()` method has to be in the `main` package. If there are several `.go` files in the folder, I think it will be even more confusing if they happen to belong to different packages.